tools/assemblystats/README_ASSEMBLY_STATS

```diff
@@ -0,0 +1,14 @@
+#Created 07/01/2011
+#Konrad Paszkiewicz, University of Exeter
+
```

Comment:
Indicate that Mariam and yourself have worked on it since the original one in 2011